build_tools/android/common/noisy_benchmarks.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""A list of noisy benchmarks and their average thresholds."""
```

Comment:
Scripts that aren't Android-specific should not put placed in a `*/android/*` folder. I can tolerate this for now given the import structure but please move them (soon).

In general we have too many scripts in weird places. These two are a frequent source of confusion for me:
* https://github.com/google/iree/blob/main/build_tools/buildkite/cmake/android/arm64-v8a/benchmark2.yml should not be Android specific
* https://github.com/google/iree/blob/main/build_tools/buildkite/cmake/android/arm64-v8a/pipeline.yml references `build_tools/kokoro/gcp_ubuntu/cmake/android/build.sh` despite not involving Kokoro at all

I find myself regularly needing to dig through layers and layers of scripts scattered across multiple overly nested directories just to see how some CMake options are set or what some python script is doing with some structured data. What should take seconds takes minutes, several times a week.

Reply:
> Scripts that aren't Android-specific should not put placed in a */android/* folder. I can tolerate this for now given the import structure but please move them (soon).

Agreed that the scripts are becoming unwieldy and mixing several things gradually. Let me do a pass afterwards to shuffle a bit and place them properly.

> In general we have too many scripts in weird places. These two are a frequent source of confusion for me:

I'm fine with whatever directory structure. That was recommended by @GMNGeoffrey in the early days. He had an opinion on how to organize them. :) We haven't changed it ever since.

Reply:
Well the directory structure with Kokoro itself was what I had opinions about. The subsequent proliferation is certainly not what I intended :grin:. I'd be interested in tackling this as part of dropping Kokoro entirely, but I don't know when we will prioritize that.